<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Ajayi Oluwaseun Emmanuel (Webdezzi Services)</author>
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>501</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>812</width>
    <height>501</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>812</width>
    <height>501</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Youtube Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/images/images/logo.png</normaloff>:/images/images/logo.png</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="videoTreeW">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>791</width>
      <height>391</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>601</width>
      <height>121</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>700</height>
     </size>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="animated">
     <bool>true</bool>
    </property>
    <attribute name="headerDefaultSectionSize">
     <number>200</number>
    </attribute>
    <column>
     <property name="text">
      <string>Video Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date Added</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>video_id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>path</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Search Video</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionReportProblem"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="ToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addURL"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="addURL">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/images/images/add.png</normaloff>:/images/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Add  URL</string>
   </property>
   <property name="toolTip">
    <string>Add Youtube URL to queue</string>
   </property>
  </action>
  <action name="actionManage_servstat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Manage service station</string>
   </property>
   <property name="toolTip">
    <string>Manage location</string>
   </property>
  </action>
  <action name="actionManage_user">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Manage users</string>
   </property>
   <property name="toolTip">
    <string>Manage user</string>
   </property>
  </action>
  <action name="actionManage_messages">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Manage tickets</string>
   </property>
   <property name="toolTip">
    <string>Manage messages</string>
   </property>
  </action>
  <action name="actionShow_left_pane">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle left pane</string>
   </property>
  </action>
  <action name="actionCreate_Service_Station">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create service station</string>
   </property>
  </action>
  <action name="actionEdit_service_station">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit service station</string>
   </property>
  </action>
  <action name="actionDelete_service_station">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete service station</string>
   </property>
  </action>
  <action name="actionHide_toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle Toolbar</string>
   </property>
  </action>
  <action name="actionManage_Logs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Manage Logs</string>
   </property>
  </action>
  <action name="actionBackup_Data">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Backup Database</string>
   </property>
  </action>
  <action name="actionManage_services">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Manage services</string>
   </property>
  </action>
  <action name="actionContact_us">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Messages</string>
   </property>
  </action>
  <action name="actionJob_application">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Job application</string>
   </property>
  </action>
  <action name="actionTraffic">
   <property name="text">
    <string>Traffic</string>
   </property>
  </action>
  <action name="actionBlocked_IPs">
   <property name="text">
    <string>Blocked IPs</string>
   </property>
  </action>
  <action name="actionReportProblem">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Report a problem</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </action>
  <action name="actionService_Station">
   <property name="icon">
    <iconset>
     <normaloff>:/images/resources/servicestation.png</normaloff>:/images/resources/servicestation.png</iconset>
   </property>
   <property name="text">
    <string>Service Station</string>
   </property>
   <property name="toolTip">
    <string>View history of service stations in your care</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionHide_toolbar</sender>
   <signal>toggled(bool)</signal>
   <receiver>ToolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
